source/codegen/templates/service_helpers.mako

```diff
@@ -192,7 +192,7 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
 % elif c_type == 'ViSession':
       auto ${parameter_name}_grpc_session = ${request_snippet};
       ${c_type} ${parameter_name} = session_repository_->access_session(${parameter_name}_grpc_session.id(), ${parameter_name}_grpc_session.name());\
-% elif c_type == 'ViInt32[]' or c_type == 'ViAddr[]':
+% elif c_type == 'ViInt32[]' or c_type == 'ViAddr[]' or c_type == 'ViUInt32[]':
```

Comment:
Change to `c_type in { 'ViAddr[]', 'ViInt32[]', 'ViUInt32[]' }`

Reply:
While you are here, can you also sort and put data types in array for line 190 above.

Reply:
changed that